pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -128,13 +133,18 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-.PHONY: docker-build
-docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+.PHONY: container-build
+container-build: test ## Build container image with the manager.
```

Comment:
/hold

I do not think that we should change the name of these targets now
It can be very disruptive and that would mean changing all docs and etc as well.
So, I would suggest you raise that in the KB + C+R meeting and then we can discuss it or try to make a pool in the channel.
Also, with this PR would come all changes in the docs